cl/search/management/commands/import_harvard_pdfs.py

```diff
@@ -86,11 +100,9 @@ def process_crosswalks(
         logger.info(
             f"Processing crosswalks. Reporter: {specific_reporter}, Resume: {resume}"
         )
-        crosswalk_dir = os.path.join(
-            os.path.dirname(os.path.dirname(os.path.dirname(__file__))),
-            "crosswalks",
+        last_reporter_file = os.path.join(
+            self.crosswalk_dir, "last_completed_reporter.txt"
         )
-        last_reporter_file = "last_completed_reporter.txt"
 
```

Comment:
why did this not work ?

Reply:
It will work, but will save the file in the current working directory instead of the crosswalk directory, i think all related files should be in the same directory